src/main/java/de/retest/ui/descriptors/Element.java

```diff
@@ -91,74 +71,22 @@ public Element( final String retestId, final IdentifyingAttributes identifyingAt
 		this.identifyingAttributes = identifyingAttributes;
 		this.attributes = attributes;
 		this.containedComponents = containedComponents;
-		this.screenshot = screenshot;
 	}
 
-	@Override
-	public int compareTo( final Element other ) {
-		final int result = identifyingAttributes.compareTo( other.getIdentifyingAttributes() );
-		if ( result != 0 ) {
-			return result;
-		}
-		return attributes.compareTo( other.getAttributes() );
-	}
-
-	@Override
-	public boolean equals( final Object obj ) {
-		if ( this == obj ) {
-			return true;
-		}
-		if ( obj == null || getClass() != obj.getClass() ) {
-			return false;
-		}
-		final Element other = (Element) obj;
-		if ( !identifyingAttributes.equals( other.identifyingAttributes ) ) {
-			return false;
-		}
-		if ( !attributes.equals( other.attributes ) ) {
-			return false;
-		}
-		return containedComponents.equals( other.containedComponents );
-	}
-
-	@Override
-	public int hashCode() {
-		if ( hashCodeCache == null ) {
-			hashCodeCache = identifyingAttributes.hashCode() + 31 * attributes.hashCode();
-		}
-		return hashCodeCache;
-	}
-
-	@Override
-	public String toString() {
-		return identifyingAttributes.toString();
-	}
-
-	public IdentifyingAttributes getIdentifyingAttributes() {
-		return identifyingAttributes;
-	}
-
-	public List<Element> getContainedElements() {
-		return containedComponents;
-	}
-
-	public boolean hasContainedElements() {
-		return !containedComponents.isEmpty();
-	}
-
-	public Screenshot getScreenshot() {
-		return screenshot;
+	public static Element withoutChildren( final String retestId, final IdentifyingAttributes identifyingAttributes,
```

Comment:
"Children" may sound intuitive, but we actually call them "contained components" (see field, getter, etc.). Therefore, I think the factory methods should do the same, e.g., `withoutContainedComponents( ... )`.

Besides this, we switched from "component(s)" to "element(s)". So, we could also think about changing the corresponding field, getter, etc. to something like `containedElements` or `childElements`.

@roesslerj?